tasks/main.yml

```diff
@@ -1,4 +1,21 @@
 ---
+- name: Update apt cache (Debian derivatives)
+  become: true
```

Comment:
Probably also needs `become_user: root` because the role may already be running under `become: true, become_user: someone_else`.